org.bible.Rosary.yaml

```diff
@@ -0,0 +1,22 @@
+id: io.github.roseblume.rosary
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: rosary
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: rosary
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 rosary /app/bin/rosary
+    - install -Dm644 applications/io.github.roseblume.rosary.desktop /app/share/applications/io.github.roseblume.rosary.desktop
+    - install -Dm644 icons/hicolor/scalable/apps/io.github.roseblume.rosary.svg /app/share/icons/hicolor/scalable/apps/io.github.roseblume.rosary.svg
+    - install -Dm644 metainfo/io.github.roseblume.rosary.xml /app/share/metainfo/io.github.roseblume.rosary.metainfo.xml
+    sources:
+      - type: archive
+        url: https://github.com/RoseBlume/Rosary/releases/download/v1.2.3/Rosary_1.2.3.zip
```

Comment:
how is this gonna work on multiple architectures? I see no compilation.

Reply:
This is intended to only be available on x86_64 and aarch64 architectures. I cannot find anything about specifiying architectures in the references. If you could guide me in the right direction that would be appreciated.

Reply:
```
$ file bin/rosary 
bin/rosary: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=5e891b159f4eeaeedf9c793d7eb3f8cb703ea39e, for GNU/Linux 3.2.0, stripped
```

how is this gonna work on aarch64

Reply:
why not building from source? There is no cross compilation necessary.

Reply:
The compiling time, and the ram usage during compilation.

Reply:
and? how is that a problem?

Reply:
It wouldn't be able to be compiled.

Reply:
explain why it wouldn't?

Reply:
The make dependencies wouldn't be able to be satisfied. It would be easier to precompile it.

Reply:
again explain why.

Reply:
It runs a rust backend and a node front end. I would need to have npm, rustup, and I would not be able to remove the gnome sdk.

Reply:
I have setup aarch64 so it should work now.

Reply:
that still doesn't explain why it wouldn't build on flathub.

Reply:
I cannot install the proper tools on flathub.

Reply:
I got it working on aarch64 so it should be fine anyways